src/ORM/PolymorphicHasManyList.php

```diff
@@ -29,6 +29,14 @@ public function getForeignClass()
         return $this->dataQuery->getQueryParam('Foreign.Class');
     }
 
+    /**
+     * Gets the field name which holds the related object class.
+     */
+    public function getForeignClassKey(): string
```

Comment:
Can we add an explicit test for this method.

Reply:
Sure thing

Reply:
Test added.